api4/system.go

```diff
@@ -545,3 +548,43 @@ func getServerBusyExpires(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 	w.Write([]byte(c.App.Srv().Busy.ToJson()))
 }
+
+func getWarnMetricsStatus(c *Context, w http.ResponseWriter, r *http.Request) {
+	if !c.App.SessionHasPermissionTo(*c.App.Session(), model.PERMISSION_MANAGE_SYSTEM) {
+		c.SetPermissionError(model.PERMISSION_MANAGE_SYSTEM)
+		return
+	}
+
+	status, err := c.App.GetWarnMetricsStatus()
+	if err != nil {
+		c.Err = err
+		return
+	}
+
+	w.Write([]byte(model.MapBoolToJson(status)))
+}
+
+func sendWarnMetricAckEmail(c *Context, w http.ResponseWriter, r *http.Request) {
+	auditRec := c.MakeAuditRecord("sendWarnMetricAckEmail", audit.Fail)
+	defer c.LogAuditRec(auditRec)
+
+	if !c.IsSystemAdmin() {
```

Comment:
Why are checking here the `IsSystemAdmin` instead of the `PERMISSION_MANAGE_SYSTEM` (the effect 99.99% of the time is going to be the same, and we prefer to avoid personalizing in roles the right to do actions).

Reply:
I agree that we should always check against the permission, though here IsSystemAdmin() is a wrapper around the PERMISSION_MANAGE_SYSTEM check, I think we should either remove it or rename it to smth. else. (better remove it imo). I removed the call to be consistent, but right now its presence feels like it causes confusion.